### ISSUE TITLE
Folders shared from a PC do not have the ignore permissions flag set

### ISSUE THREAD

**Issue by lukash** [syncthing/syncthing-android#821]
If I create a new folder on a PC and share it with an Android device, after tapping on the notification I get on Android and creating the folder, the sync fails with:

`13:54:14 INFO: Puller: final: chmod /storage/emulated/0/FOLDER/FILE: operation not permitted`

Looking at #146, I checked in the WebUI and the IgnorePermissions flag is not set on such a folder, whereas it is set on a folder created through the Android client.

    App Version: 0.9.2
    Syncthing Version: v0.14.14
    Android Version: Android 6.0.1
    Remote Desktop Syncthing Version: v0.14.{14,15}

**Comment by lukash** [comment]
Ok, I just created another folder, this time on the phone. It had some initial contents. Shared it with a PC. Then linked the folder on a PC, setting it to a directory which also had some prior contents, different from the phone one.

It synced to the PC fine, but didn't sync the files to the phone. Turns out that folder also didn't have the Ignore Permissions flag set...

Furthermore, this time the log was empty except for the first line:
`--------- beginning of main`

No errors...... After I fixed the IgnorePermissions flag in the WebUI, it filled with a few regular log lines, while the folder synced up fine.

Really, I'm at a loss with this app......
